javascript/ql/src/Security/CWE-094/ExpressionInjection.qhelp

```diff
@@ -12,16 +12,21 @@
 
 		</p>
 
+        <p>
+            Code injection in GitHub actions may allow an attacker to 
+            exfiltrate the temporary GitHub repository authorization token. 
+            The token has write access to the repository, and thus an attacker
+            can use it to modify the repository.
```

Comment:
The GITHUB_TOKEN has write access by default, but it is configurable today: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token.

Examples: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#example-1-passing-the-github_token-as-an-input

Reply:
I've tried to update the QHelp such that it should be clear that the GITHUB_TOKEN only sometimes has write access.